packages/bierzo-wallet/src/setupTests.js

```diff
@@ -0,0 +1,4 @@
+export const window = global.window || {};
+
+global.window = window;
+global.config = require('../build/static/config/conf.json');
```

Comment:
What about always loading the `./config/development.json` from the source repo? Tests will always need this one. This way, the tests can run undependent on the `./build` directory (at least unit and dom tests).

Reply:
ok, no problems